### ISSUE TITLE
Battery shows too many decimals

### ISSUE THREAD

**Issue by ioanmoldovan95** [Ordissimo/Big-Sur-StatusArea#61]
System: Thinkpad T480
OS: Fedora 35
Gnome Version: 41.3, Wayland

Most probably this happens on laptops with a dual-battery system. 
The battery percentage has too many decimals and takes a lot of space in the top bar. It should be limited to at most 2 decimals.

**Comment by Ordissimo** [comment]
Hi @ioanmoldovan95 

Can you modify line 80 of this file ~/.local/share/gnome-shell/extensions/Big-Sur-StatusArea/indicators/power.js
```
this._percentageLabel.clutter_text.set_markup('<span size="smaller">' + this._power._proxy.Percentage + " %</span>");
# to
this._percentageLabel.clutter_text.set_markup('<span size="smaller">' + this._power._proxy.Percentage.toFixed(2) + " %</span>");
```
Give me a feedback

**Comment by Ordissimo** [comment]
Fixed in #69 
Thank's
